system/src/main.rs

```diff
@@ -11,8 +11,7 @@
 #![cfg_attr(not(test), no_main)]
 #![cfg_attr(not(test), no_std)]
 
+libhypatia::define_segment!();
+
 #[start]
 pub extern "C" fn init() {}
```

Comment:
(ironically `system` is a task, not a segment, and _should_ have `main` and not `init`. I'll fix that up later.)